config/routes.rb

```diff
@@ -208,7 +208,7 @@
   get 'help/:topic' => 'help#faq'
 
   #Protocol Url
-  get 'protocol' => redirect("http://wiki.diasporafoundation.org/Federation_Protocol_Overview")
+  get 'protocol' => redirect("https://wiki.diasporafoundation.org/Federation_Protocol_Overview")
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.